cmd/netstate-http/main.go

```diff
@@ -30,6 +31,14 @@ func initializeFlags() {
 	flag.Parse()
 }
 
+// server will pull from these creds and compare with client creds
+// will switch to macroons later
+func setEnv() {
+	viper.SetEnvPrefix("API")
+	os.Setenv("API_KEY", "abc123")
```

Comment:
is this supposed to be doing this or is it left over from testing?

Reply:
it's suppose to do this, for now; After i work on the auth for the GRPC server, I'll be switching to macaroons altogether.  @jtolds can confirm.

Reply:
you actually don't need the os.Setenv line. ideally, when you start your program, you'll have already run `export API_KEY=abc123` in bash, so you don't need this here